pkg/cmd/package/upload/upload.go

```diff
@@ -175,11 +195,42 @@ func uploadRun(cmd *cobra.Command, f factory.Factory, flags *UploadFlags) error
 				case constants.OutputFormatBasic:
 					cmd.Printf("%s\n", path)
 				default:
-					if created {
+					if uploadResult.CreatedNewFile {
 						cmd.Printf("Uploaded package %s\n", path)
 					} else {
 						cmd.Printf("Ignored existing package %s\n", path)
 					}
+
+					if uploadResult.UploadMethod == packages.UploadMethodDelta && uploadResult.UploadInfo != nil {
+						deltaInfo := uploadResult.UploadInfo
+						deltaRatio := 0.0
+						if deltaInfo.FileSize > 0 {
+							deltaRatio = float64(deltaInfo.DeltaSize) / float64(deltaInfo.FileSize) * 100
+						}
+
+						switch deltaInfo.DeltaBehaviour {
+						case packages.DeltaBehaviourNoPreviousFile:
+							cmd.Printf("    Full upload for package %s. No previous versions available\n"+
+								"    Timing: Signature %v, Upload %v\n",
+								path, roundDuration(deltaInfo.RequestSignatureDuration), roundDuration(deltaInfo.UploadDuration))
+						case packages.DeltaBehaviourNotEfficient:
+							cmd.Printf("    Full upload for package %s. Delta size was %.1f%% of full file (too large)\n"+
+								"    Timing: Signature %v, Build Delta %v, Upload %v\n",
+								path, deltaRatio,
+								roundDuration(deltaInfo.RequestSignatureDuration), roundDuration(deltaInfo.BuildDeltaDuration), roundDuration(deltaInfo.UploadDuration))
+						case packages.DeltaBehaviourUploadedDeltaFile:
+							cmd.Printf("    Delta upload for package %s.\n"+
+								"    Delta size was %.1f%% of full file, saving %d bytes\n"+
```

Comment:
I like this output, would it be more useable with human readable numbers?
e.g in my testing I got `Delta size was 0.0% of full file, saving 252737468 bytes`

Reply:
I've wrapped it with the existing `humanReadableBytes` helper we have in the CLI